docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -1,40 +1,18 @@
 [[ingest-management-troubleshooting]]
 [role="xpack"]
-= Troubleshooting
+= Troubleshoot common problems
 
 beta[]
 
-We have collected the most common known problems and frequently asked questions
-here. If your question isn't answered here, please review open issues in the
-following GitHub repositories:
+We have collected the most common known problems here. If your problem isn't
+described here, please review open issues in the following GitHub repositories:
 
 * https://github.com/elastic/kibana/issues[{kib}]
 * https://github.com/elastic/beats/issues[{beats}]
 * https://github.com/elastic/package-registry/issues[{package-registry}]
 
```

Comment:
do we want to do some cross linking here?  ... we could put in the new public endpoint repo:
https://github.com/elastic/endpoint/issues